app/socket-apis/get-points-of-ids.js

```diff
@@ -0,0 +1,64 @@
+// https://github.com/EnCiv/civil-pursuit/issues/203
+const Points = require('../models/points')
+const { ObjectId } = require('mongodb')
+
+async function getPointsOfIds(ids, callback) {
+  const cbFailure = errorMsg => {
+    if (errorMsg) console.error(errorMsg)
+    if (callback) callback({ points: [], myWhys: [] }) // Return empty arrays on error
```

Comment:
If there is an error, cb(), so that the call back received undefined. Empty props means that nothing was found, which is a success.